src/containers/TextInputContainer.js

```diff
@@ -36,7 +36,6 @@ const Label = styled.Text `
   font-family: ${theme.FONT_REGULAR};
   line-height: ${theme.SPACING_SLIGHT_MEDIUM};
   color: ${(props) => (props.textColor ? props.textColor : theme.COLOR_BLACK)};  
-  margin-top: 10;
```

Comment:
is there a reason why you removed this? TextInputContainer is also used in UserProfile so it might affect that screen.

Reply:
I checked UserProfile and nothing was changed.  Margin-top was removed because placeholder was positioned too high compared to how it is presented in the prototype.

Reply:
@nouj Just checked Profile screen and it is affecting the margin on that specific field.

Reply:
![E08D4A27-7530-4FB0-9DB5-750C5239C753](https://user-images.githubusercontent.com/72771523/146100338-717df28f-5e9a-4aa3-aefb-28e6ddd4fb1e.jpeg)